### DISCUSSION TITLE
How do you create an Azure Application ?

### DISCUSSION THREAD

**Discussion by tonyjameshart** [Venafi/venafips#100] [Q&A] [answered]
How do you create an Azure Application ?
Tried   
$path = "\VED\Policy\Path to \Device"  
and with a final slash.  $path = "\VED\Policy\Path to \Device\"   
The error says "ObjectInvalidContainment"   The path is to the Device.  Is the Path to include the name of the app to be?

New-TppObject  -Class 'Azure Key Vault' -Attribute @{'Driver Name'='appazurekeyvault';'Vault Name'='"$VaultName"} -Path "$path" -VenafiSession $VenafiSession 

What am I doing wrong?

**Comment by gdbarron** [comment, marked as answer]
Hi @tonyjameshart.  The path should be to the object itself, not the containing device.  Everything else looks good.

<img width="789" alt="image" src="https://user-images.githubusercontent.com/11862024/163384289-6c58aeae-6dc9-429f-8181-480c967c0bc7.png">

Function help is your friend :)
